Sources/Shared/Models/VIMConnection.m

```diff
@@ -83,4 +83,12 @@ - (BOOL)canPost
     return (self.options && [self.options containsObject:@"POST"]);
 }
 
+- (void)didFinishMapping
```

Comment:
Now that `VIMConnection` has a `-didFinishMapping` method, we'll want to make sure that we call up to `super` from `VIMNotificationsConnection`'s implementation (and any other subclass implementations, but I think that's the only one).